notarymysql/start

```diff
@@ -93,6 +121,14 @@ if [ -n "${DB_USER}" -o -n "${DB_NAME}" ]; then
     sleep 1
   done
 
+  # To check whether the old database is exist or not and warning people to
```

Comment:
rewording nit: "Check whether the old database exists and warn users to manually migrate those tables if so"